src/Components/PresentationalComponents/CVEDetailsPageDescription/CVEDetailsPageDescription.js

```diff
@@ -17,17 +17,33 @@ const CVEDetailsPageDescription = ({ cveAttributes, intl }) => {
         <CVEPageContext.Consumer>
             {context =>
                 <Stack hasGutter>
-                    <StackItem className="pf-u-mt-sm">
-                        <div>
-                            <span className="pf-u-mr-xs">{intl.formatMessage(messages.publishDate)}:</span>
-                            <WithLoader
-                                isLoading={context.isLoading}
-                                variant={LoaderType.inlineSkeleton}
-                                style={{ width: '100px' }}
-                            >
-                                {publishDate}
-                            </WithLoader>
-                        </div>
+                    {
+                        publishDate ?? (
+                            <StackItem className="pf-u-mt-sm">
+                                <div>
+                                    <span className="pf-u-mr-xs">{intl.formatMessage(messages.publishDate)}:</span>
+                                    <WithLoader
+                                        isLoading={context.isLoading}
+                                        variant={LoaderType.inlineSkeleton}
+                                        style={{ width: '100px' }}
+                                    >
+                                        {publishDate}
+                                    </WithLoader>
+                                </div>
+                            </StackItem>
+                        )
+                    }
+                    <StackItem>
+                        <WithLoader
+                            isLoading={context.isLoading}
+                            variant={LoaderType.rectangle}
+                            style={{ height: '132px', width: '100%' }}
+                        >
+                            <TextContent style={{ textAlign: 'justify' }}>
+                                <Text component={TextVariants.p}>{description ?
+                                    description : intl.formatMessage(messages.noDescription)}</Text>
```

Comment:
This can be simplified to `{description || intl.formatMessage(messages.noMetadataDescription)}`